Plugin.pm

```diff
@@ -1390,29 +1399,8 @@ sub QobuzGetTracks {
 				push @$items, $item;
 			}
 
-			# Build a consolidated list of all artists on the album
-			my @uniquePerformers;
-			if ( scalar @performers ) {
-				my %seen = ();
-				my $tracks;
-				foreach my $item (@performers) {
-					push @{$tracks->{$item->{'name'}}->{'tracks'}}, $item->{'track'};
-					delete $item->{'track'};
-					push(@uniquePerformers, $item) unless $seen{$item->{'name'}}++;
-				}
-				foreach my $item (@uniquePerformers) {
-					my @tracks = @{$tracks->{$item->{'name'}}->{'tracks'}};
-					if ( @tracks && scalar @tracks < $album->{tracks_count} ) {
-						$item->{'name'} .= scalar @tracks == 1 ? " (" . cstring($client, 'PLUGIN_QOBUZ_TRACK_LC') . " " : " (" . cstring($client, 'PLUGIN_QOBUZ_TRACKS_LC') . " ";
-						$item->{'name'} .= join(", ", @tracks) . ")";
-					}
-				}
-				my $item = {
-					name => cstring($client, 'PLUGIN_QOBUZ_PERFORMERS'),
-					items => \@uniquePerformers,
-				};
-				push @$items, $item;
-			}
+			# Add a consolidated list of all artists on the album
+			$items = _albumPerformers($client, $performers, $album->{tracks_count}, $items);
```

Comment:
This would override/remove any previously added items, as you're replacing all of `$items` instead of adding to it.

Reply:
Got it, thanks!